pkg/ccl/changefeedccl/sink_cloudstorage.go

```diff
@@ -663,6 +673,27 @@ func (s *cloudStorageSink) flushFile(ctx context.Context, file *cloudStorageSink
 	}
 	s.asyncFlushActive = asyncFlushEnabled
 
+	if file.parquetCodec != nil {
```

Comment:
Let's move the body of this function into a helper (`flushParquet` or some such)